ansible/roles/controller/templates/chainlet/chainlet-deploy.yml.tmpl.j2

```diff
@@ -135,10 +203,17 @@ spec:
           resources:
             %%% controller_chainlet_resources | to_nice_yaml(indent=2) | indent(12, true) | trim %%%
           volumeMounts:
-            - name: chainlet-volume
+            - name: "saga-{{ .Name }}-persistent-storage"
               mountPath: "/root/.sagaosd"
               subPath: sagaosd
+            - name: "fluentd-logs"
+              mountPath: "/var/log/saga"
+              subPathExpr: $(UID)
       volumes:
-        - name: "chainlet-volume"
+        - name: "saga-{{ .Name }}-persistent-storage"
           persistentVolumeClaim:
-            claimName: chainlet-pvc
+            claimName: "saga-{{ .Name }}-pvc"
+        - name: "fluentd-logs"
```

Comment:
Fluentd is broken and won't work with our amount of logs times hundreds of chains. That's why I removed it. This should at least be optional. Did you intentionally re-added it? Or did you copy/paste from `node-infra`